client/src/components/ImageContainer.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+function ImageContainer({ isLarge, movie }) {
+  const baseUrl = 'https://image.tmdb.org/t/p/original/';
+
+  return (
+    <>
+      <MovieContainer netflixStyle={isLarge}>
+        <ImageStyler
+          netflixStyle={isLarge}
+          key={movie.id}
+          src={`${baseUrl}${isLarge ? movie.poster_path : movie.backdrop_path}`}
+          alt={movie.name}
+        />
+        <Overlay></Overlay>
+        {/*Add to Watchlist icon will be placed in Overlay*/}
+      </MovieContainer>
+    </>
+  );
+}
+
+export default ImageContainer;
+
+const MovieContainer = styled.div(
+  (props) => css`
+    position: relative;
+    margin-right: 10px;
+    transition: transform 450ms;
+    width: 100%;
+    :hover {
+      transform: scale(1.08);
+    }
+    :hover div {
+      opacity: 1;
+    }
+    ${props.netflixStyle &&
+    css`
+      :hover {
+        transform: scale(1.09);
+      }
+    `}
+  `
+);
+
+ImageContainer.propTypes = {
+  isLarge: PropTypes.bool,
+  movie: PropTypes.object,
+  baseUrl: PropTypes.string,
+};
```

Comment:
Consider putting the propTypes either above or below the styled components so that you have all your styling in one block. I was a bit confused to see the PropTypes and then styling again.

Reply:
Good catch, I intended the same but missed that one ;)